client/src/components/DiceRoller.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {ThemeProvider, theme, CSSReset, Flex, Box, Heading, IconButton, Link, FormControl, FormLabel, Input, Stack, Checkbox, Button} from "@chakra-ui/react";
```

Comment:
It seems like the following classes are not used: IconButton, Link, FormControl, FormLabel, Input, Stack, Checkbox, Button

Are they necessary?

Reply:
Not necessary as they are not used, removed